### ISSUE TITLE
Organisations can be hyper linked to org's home page

### ISSUE THREAD

**Issue by a0xnirudh** [amfoss/club-website-2017#26]
It would be nice if we can hyper link the organisation name in the contribution section to corresponding org's homepage.

**Comment by tonythomas01** [comment]
I dont think that can be fixable though :( You will have to take that link as an input in the acheivment box - store it in a db, and later find add it as an <a> tag to the organization name. 
I wont rate it do-able at this point.

**Comment by anjanasasindran123** [comment]
@tonythomas01We will just see. if not possible/high risk, then leave it.
Alright!

**Comment by ash1794** [comment]
What's the DB backend that is being used? If we are using a nosql db like mongodb it'd be fairly easy to make the modification. Otherwise it'll be a headache.
